frontend/src/components/maps/MineMapLeaflet.js

```diff
@@ -213,6 +230,67 @@ class MineMapLeaflet extends Component {
     });
   };
 
+  addLatLongCircle = () => {
+    if (this.props.lat && this.props.long && !this.props.mineName) {
+      L.circle([this.props.lat, this.props.long], {
+        color: "red",
+        fillColor: "#f03",
+        fillOpacity: 0.25,
+        radius: 500,
+        stroke: false,
+      }).addTo(this.map);
+    }
+  };
+
+  addMinePinClusters = async () => {
```

Comment:
Any intended reason why this is an async function? There's no `await` here and no returned value.